### PR TITLE
adds a page for each individual team member

### PR DESCRIPTION
**Pull request by ColinW520**
Addresses #87 #88 

#### This PR introduces a new page for to show each team member individually.

It can be accessed by going to /team/#{team_member.name}

Here are the mockups:

https://spectrumtrail.invisionapp.com/overview/Race-Team-Pages-ck3yuuocs0pvv017ok220s1po/screens?v=Kudu%2BqqasENdl6DEeaU%2FCA%3D%3D&linkshare=urlcopied

Digressions from the mockups:
 - use a white background
 - move title text to image cover
 - add 'profile' button to team member cards

#### How to review this PR:

 - [ ] Pull down this branch
 - [ ] Run `rake db:migrate`
 - [ ] Restart `rails s`
 - [ ] Fill in necessary data for each team member
 - [ ] Navigate to each team member's page

### CODE REVIEW

**Comment by willblake01** [comment]
We do still have the member card filling the entire row after wrapping. Should we open an issue for that?
Screen Shot 2019-12-13 at 9 08 03 AM

File: app/views/team_members/_leader.html.haml [hunk 1]
```diff
@@ -4,3 +4,10 @@
     %h4.card-title.text-white= team_member.name
     %hr.bg-white
     %h6.card-title.text-white= team_member.title
+    - if team_member.best_advice?
+      %q.card-text.text-white= team_member.best_advice
+    %br
+  .card-footer.bg-tertiary
+    %div.text-center
+      = link_to team_member_path(team_member), class: "btn btn-md border-white text-white bg-dark" do
+        PROFILE
```
[the remark this answers is not in the document]
Comment (ColinW520) [reply app/views/team_members/_leader.html.haml]: ^ correct

**Comment by ColinW520** [comment]
> We do still have the member card filling the entire row after wrapping. Should we open an issue for that?

^ This will only happen if we have an odd number of team members. I am looking into adding some handling for that. For now, most importantly, it is responsive.

**Comment by ColinW520** [comment]
> We do still have the member card filling the entire row after wrapping. Should we open an issue for that?

^ additionally, those changes are not related to this branch, as they were already committed.

File: app/views/admin/team_members/_form.html.haml [hunk 2]
```diff
@@ -3,45 +3,76 @@
...
+              %small.text-muted Please select exatly one
+            - if f.object.cover_image.attached?
+              .attachment-wrapper{id: "attachment-wrapper-#{f.object.cover_image.id}"}
+                = image_tag f.object.cover_image,
+                  class: "img img-fluid img-thumbnail"
+                = link_to attachment_path(id: f.object.cover_image.id),
+                  method: :delete,
+                  remote: true,
+                  class: "text-danger" do
+                  %i.fa.fa-times
```
[the remark this answers is not in the document]
Comment (ColinW520) [reply app/views/admin/team_members/_form.html.haml]: This was for spacing/readability

File: app/views/team_members/show.html.haml [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+            %h4.h4 Keep up with #{@team_member.name}!
+      .row
+        .col-12
+          .text-center
+            - if @team_member.strava_link.present?
+              %a{href: @team_member.strava_link, target: "_blank"}
+                %i.fab.fa-strava.fa-4x.text-warning.mr-3
+            - if @team_member.instagram_link.present?
+              %a{href: @team_member.instagram_link, target: "_blank"}
+                %i.fab.fa-instagram.fa-4x.text-tertiary
```
[the remark this answers is not in the document]
Comment (ColinW520) [reply app/views/team_members/show.html.haml]: ^ not sure what you're referencing here as a restful action?

**Pull request merged**